packages/xod-arduino/platform/formatNumber.h

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
Look at the resulting C++ for `blink.cpp`. I suggest prefixing any inlined source file with the following banner to be consistent:

```cpp
/*=============================================================================
 *
 *
 * Bla bla bla bla
 *
 *
 =============================================================================*/
```

Otherwise, the structure is lost a bit